web/src/app/alerts/AlertsList.js

```diff
@@ -233,6 +214,25 @@ export default function AlertsList(props) {
     }
   }
 
+  /*
+   * Gets the header to display above the list to give a quick overview
+   * on if they are viewing alerts for all services or only their
+   * favorited services
+   */
+  function getHeaderNote() {
+    const { favoritesOnly, includeNotified } = variables.input
+
+    if (includeNotified && favoritesOnly) {
+      return 'Showing alerts you have been notified of and from any services you have favorited.'
+    }
+
+    if (allServices) {
+      return 'Showing alerts for all services.'
+    }
+
+    return ''
```

Comment:
What case would this be? If it's not expected, maybe throw an error here, or at least log one.

Reply:
Currently just a default, which is used on the services alerts pages. If an unknown filter comes in (or one that we are not covering on purpose), we don't display anything. In the case of the services list, we only care about displaying the alerts for that service (no "all services" or "my alerts" to worry about distinguishing between), so we can leave the note blank

Reply:
On the service alert page, it wouldn't be a bad idea to display "Displaying alerts for Foo Bar." or something like that. When we did UX testing, it came up that it was hard to tell the difference between the two alert pages -- always having a filter string could help with that.

Also it might be good to include the status filter in that string as well, especially on mobile where the headers are not rendered. Again following the newer convention of always describing the current filter set. (e.g. schedule shifts).